packages/os/autoscaling-warm-pool.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Run tasks reliant on IMDS
+After=network-online.target settings-applier.service
+Requires=settings-applier.service
+Wanted=network-online.target
+Before=kubelet.service
```

Comment:
`kubelet.service` needs to take the dependency on the "wait for in service" service, not the other way around - this service provides more general functionality.